src/collections/_documentation/error-reporting/getting-started-verify/csharp.md

```diff
@@ -1,3 +1,5 @@
+What better way to send your first event than to break your application!
```

Comment:
```suggestion
```

I'd remove this sentence because the sentence after communicates the same thing, but more specifically.